[2025-03-19T18:40:49.314+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T18:40:49.445+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2023-10-02T00:00:00+00:00 [queued]>
[2025-03-19T18:40:49.579+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2023-10-02T00:00:00+00:00 [queued]>
[2025-03-19T18:40:49.730+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T18:40:49.759+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2023-10-02 00:00:00+00:00
[2025-03-19T18:40:49.880+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=505) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T18:40:49.881+0000] {standard_task_runner.py:72} INFO - Started process 545 to run task
[2025-03-19T18:40:49.810+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v1', 'get_name', 'scheduled__2023-10-02T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpz_h3uqf0']
[2025-03-19T18:40:49.900+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask get_name
[2025-03-19T18:40:51.082+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2023-10-02T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T18:40:51.902+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v1' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-02T00:00:00+00:00'
[2025-03-19T18:40:51.916+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T18:40:51.934+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2025-03-19T18:40:52.106+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T18:40:52.117+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v1, task_id=get_name, run_id=scheduled__2023-10-02T00:00:00+00:00, execution_date=20231002T000000, start_date=20250319T184049, end_date=20250319T184052
[2025-03-19T18:40:52.283+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T18:40:52.349+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T18:40:52.357+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
